diff --git a/setup.py b/setup.py
index 3310597c..216f6571 100644
--- a/setup.py
+++ b/setup.py
@@ -184,7 +184,9 @@ def get_openmp_flag(compiler):

 def check_openmp_support():
     """Check whether OpenMP test code can be compiled and run"""
-
+    # Short circuit OpenMP check while cross compiling on conda-forge
+    if os.environ.get("CONDA_BUILD_CROSS_COMPILATION") == "1":
+        return True
     try:
         from setuptools._distutils.ccompiler import new_compiler
         from setuptools._distutils.sysconfig import customize_compiler